sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -1021,6 +1021,18 @@ class ParquetFilterSuite extends QueryTest with ParquetTest with SharedSQLContex
       }
     }
   }
+
+  test("SPARK-25206: wrong records are returned when Hive metastore schema and parquet schema " +
```

Comment:
this is a end-to-end test and should not be put here. How about `HiveParquetSuite`